nucliadb/nucliadb/standalone/purge.py

```diff
@@ -20,7 +20,13 @@
 # Standalone purge command
 import asyncio
 
-import pydantic_argparse
+try:
+    import pydantic.v1 as pydantic
+    import pydantic2_argparse as pydantic_argparse
```

Comment:
Here aswell: you can assume that the nucliadb package version will come with pydantic2_argparse from now on, no need to keep backwards compatibility.

Just make sure to set the right version dependencies in the `requirements.txt` and `requirements.lock.txt`